pkg/settings/settings.go

```diff
@@ -29,9 +29,7 @@ var (
 	SupportBundleImage           = NewSetting("support-bundle-image", "rancher/support-bundle-kit:v0.0.3")
 	SupportBundleImagePullPolicy = NewSetting("support-bundle-image-pull-policy", "IfNotPresent")
 	DefaultStorageClass          = NewSetting("default-storage-class", "longhorn")
-	CpuOvercommitPercentage      = NewSetting("cpu-overcommit-percentage", "1000")    // KubeVirt's default is "10"x ratio. To align with others, we change it to percentage.
-	MemoryOvercommitPercentage   = NewSetting("memory-overcommit-percentage", "100")  // KubeVirt's default
-	StorageOvercommitPercentage  = NewSetting("storage-overcommit-percentage", "200") // Longhorn's default
+	OvercommitConfig             = NewSetting("overcommit-config", `{"cpu":1000,"memory":100,"storage":200}`) // KubeVirt & Longhorn's defaults
```

Comment:
I would expect the default overcommit of both CPU and memory will be 200%, also can we do the ratio calculation in the method and keep the setting config here as a percentage only.

Reply:
Sorry but didn't get the point of "do the ratio calculation in the method". In which method do you mention to do the calculation? 

> keep the setting config here as a percentage only.

Aren't we already use percentage here? Thanks.

Reply:
Updated all defaults to 200% and rebased.

Reply:
@guangbochen Any reason for the default values? The 200% memory default looks aggressive to me. FYI openstack use 16:1 for CPU and 1.5:1 for memory as the default: https://docs.openstack.org/arch-design/design-compute/design-compute-overcommit.html

Reply:
I think the referenced doc is more meaningful in regards to VM's overcommit, @weihanglo can u PTAL and this will need a doc PR of the new changes, thanks.

Reply:
Just rebased and updated to 16:1 for CPU and 1.5:1 for memory. 

The number looks reasonable to me since CPU overcommitment is not that harmful than out-of-memory. KubeVirt also defaults to a similar large ratio such as 10:1 for CPU. Thanks.